CRAWLR/SearchCell.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  SearchCell.swift
+//  CRAWLR
+//
+//  Created by Rachel Bright on 4/23/20.
+//  Copyright © 2020 Wesley Swanson. All rights reserved.
+//
+
+import UIKit
+
+class SearchCell: UICollectionViewCell {
+    
+    @IBOutlet weak var picture: UIImageView!
+    @IBOutlet weak var name: UILabel!
+    @IBOutlet weak var distance: UILabel!
+    @IBOutlet weak var address: UILabel!
+    @IBOutlet weak var status: DesignableLabel!
```

Comment:
This isn't being used and should be removed.

Reply:
Do you mean just the status label?